src/config/data/sampleData.js

```diff
@@ -0,0 +1,79 @@
+const sampleData = [
+    {
+        "user_id": 1,
```

Comment:
Hey! This can be removed. I think client side I may mock more data and I'd need the user id to mock sessionid (probably not, but going to be kept for now)